### ISSUE TITLE
파이토치_연습1

### ISSUE THREAD

**Issue by yuseungwoo** [yuseungwoo/pytorch#1]
#bhttps://github.com/bjpublic/PyTorch

import torch
torch.manual_seed(70)
import torch.nn as nn

def identity_func(x):
    return x

linear_ih = nn.Linear(in_features=2, out_features=2) 
linear_ho = nn.Linear(in_features=2, out_features=1)
activation_layer = nn.Sigmoid()

x = torch.Tensor([[0, 1]])
z1 = linear_ih(x)
a1 = activation_layer(z1)
z2 = linear_ho(a1)
y = identity_func(z2)

print(y.item())
